@navikt/core/react/src/collapsible/parts/Collapsible.Trigger.tsx

```diff
@@ -32,7 +36,7 @@ export const CollapsibleTrigger = forwardRef<
       {...rest}
       id={ctx.triggerId}
       aria-controls={ctx.open ? ctx.contentId : undefined}
-      aria-expanded={ctx.open}
+      aria-expanded={disableAriaExpanded ? undefined : ctx.open}
```

Comment:
Morten var enig i at aria-expanded kunne droppes siden teksten på knappen endres. Hva synes du om denne løsningen?

Reply:
Tror jeg liker bedre at man sender inn `aria-expanded={undefined}` i consumer og da er før `...rest` her. Men siden dette er et internt API for nå funker denne varianten også 👍

Reply:
Jeg var veldig usikker på hva som var best. Ditt forslag er cleanere. Men bør man kunne sette til true eller false fra utsiden også? Hvis ikke, bør det vel funke å sette typen til `"aria-expanded"?: undefined;`